src/Uno.UWP/System/Power/PowerManager.iOS.cs

```diff
@@ -0,0 +1,150 @@
+﻿#if __IOS__
+using System;
+using UwpBatteryStatus = Windows.System.Power.BatteryStatus;
+using UwpEnergySaverStatus = Windows.System.Power.EnergySaverStatus;
+using UwpPowerSupplyStatus = Windows.System.Power.PowerSupplyStatus;
+using UIKit;
+using Foundation;
+
+namespace Windows.System.Power
+{
+	public partial class PowerManager
+	{
+		private const string DeviceModelSimulator = "iPhone Simulator";
+
+
+		private static NSObject _batteryLevelChangeSubscription;
+		private static NSObject _batteryStateChangeSubscription;
+		private static NSObject _powerStateChangeSubscription;
+
+		private static readonly UIDevice _device;
+		private static readonly bool _isSimulator;
+
+		static PowerManager()
+		{
+			_device = UIDevice.CurrentDevice;
+			_isSimulator = _device.Model == DeviceModelSimulator;
+			_device.BatteryMonitoringEnabled = !_isSimulator;
+		}
+			   
+		private static void StartEnergySaverStatusMonitoring()
+		{
+			if (_isSimulator) return;
+			TryAddNotificationSubscription(
+				NSProcessInfo.PowerStateDidChangeNotification,
+				ref _powerStateChangeSubscription,
+				PowerStateChangeHandler);
+		}
+
+		private static void EndEnergySaverStatusMonitoring() =>
+			TryRemoveNotificationSubscription(ref _powerStateChangeSubscription);
+
+		private static void StartRemainingChargePercentMonitoring()
+		{
+			if (_isSimulator) return;
+			TryAddNotificationSubscription(
+				UIDevice.BatteryLevelDidChangeNotification,
+				ref _batteryLevelChangeSubscription,
+				BatteryLevelChangeHandler);
+		}
+
+		private static void EndRemainingChargePercentMonitoring() =>
+			TryRemoveNotificationSubscription(ref _batteryLevelChangeSubscription);
+
+		private static void StartPowerSupplyStatusMonitoring() =>
+			StartIosBatteryStateMonitoring();
+
+		private static void EndPowerSupplyStatusMonitoring() =>
+			EndIosBatteryStateMonitoring();
+
+		private static void StartBatteryStatusMonitoring() =>
+			StartIosBatteryStateMonitoring();
+
+		private static void EndBatteryStatusMonitoring() =>
+			EndIosBatteryStateMonitoring();
+
+		private static void StartIosBatteryStateMonitoring()
+		{
+			if (!_isSimulator) return;
```

Comment:
```suggestion
			if (_isSimulator) return;
```

Reply:
Yes, refactor typo... I need to retest iOS again.